### PR TITLE
Feature/nijo 332 voucher content

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/Models/Mailing/BookingVoucherData.cs [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
-    public class BookingVoucherData
+    public readonly struct BookingVoucherData
     {
-        public string BookingId { get; set; }
-        public string CheckInDate { get; set; }
-        public string CheckOutDate { get; set; }
-        public List<BookingRoomDetails> RoomDetails { get; set; }
-        public string AccommodationName { get; set; }
-        public string ReferenceCode { get; set; }
+        public BookingVoucherData(int bookingId, in AccommodationInfo accommodation,
```
Comment (kirillta) [inline Api/Models/Mailing/BookingVoucherData.cs:10]: Think we need remarks and policies in a voucher

File: Api/Services/Mailing/BookingMailingService.cs [hunk 2]
```diff
@@ -23,20 +22,16 @@ public class BookingMailingService : IBookingMailingService
         }
 
 
-        public Task<Result> SendVoucher(int bookingId, string email)
-        {
-            return SendEmail(email,
+        public Task<Result> SendVoucher(int bookingId, string email, string languageCode)
+            => SendEmail(email,
```
Comment (kirillta) [inline Api/Services/Mailing/BookingMailingService.cs:26]: I suggest to extend the method and able to send a mуыыфпу to multiple emails

File: Api/Services/Accommodations/Bookings/BookingDocumentsService.cs [hunk 3]
```diff
@@ -1,69 +1,122 @@
...
+                if (!_bankDetails.AccountDetails.TryGetValue(serviceDetails.RoomContractSet.Price.Currency, out var accountData))
+                    _bankDetails.AccountDetails.TryGetValue(Currencies.USD, out accountData);
+                
+                var sellerDetails = new BookingInvoiceData.SellerInfo(_bankDetails.CompanyName, _bankDetails.BankName, _bankDetails.BankAddress,
+                    accountData.AccountNumber, accountData.Iban, _bankDetails.RoutingCode, _bankDetails.SwiftCode);
+
+                //TODO: add a contract number and a billing email after company table refactoring
+                var buyerDetails = new BookingInvoiceData.BuyerInfo(company.Name, company.Address, "contractNumber", "billingEmail");
+
+                var roomDetails = new List<BookingRoomDetails>();
```
Comment (kirillta) [inline Api/Services/Accommodations/Bookings/BookingDocumentsService.cs:109]: Empty details so far

File: Api/Services/Mailing/BookingMailingService.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Mailing/BookingMailingService.cs:26]: Will fix that in further PRs

File: Api/Services/Accommodations/Bookings/BookingDocumentsService.cs (hunk 3 above)
Comment (mingazhev) [reply Api/Services/Accommodations/Bookings/BookingDocumentsService.cs:109]: Will fix that in further PRs

**Pull request merged**